com.musala.simple.students.spring/src/main/resources/test.txt

```diff
@@ -0,0 +1 @@
+This is a test.
```

Comment:
What do we need this file for? :) If it's unused, please remove it here and in the other project (I haven't checked if it's present there too). ;)

Reply:
I use this file in the FileHelperTest class to test the readFile method :?

Reply:
Fine, my bad ;)